src/Control/Effect/Exception.hs

```diff
@@ -0,0 +1,254 @@
+{-# LANGUAGE DeriveFunctor, ExistentialQuantification, RankNTypes, StandaloneDeriving #-}
+-- | Operations from "Control.Exception" lifted into effectful contexts using 'Control.Effect.Lift.Lift'.
+--
+-- @since 1.0.0.0
+module Control.Effect.Exception
+( -- * Lifted "Control.Exception" operations
+  throwIO
+, ioError
+, throwTo
+, catch
+, catches
+, Handler(..)
+, catchJust
+, handle
+, handleJust
+, try
+, tryJust
+, evaluate
+, mask
+, mask_
+, uninterruptibleMask
+, uninterruptibleMask_
+, getMaskingState
+, interruptible
+, allowInterrupt
+, bracket
+, bracket_
+, bracketOnError
+, finally
+, onException
+, module Control.Exception
+  -- * Lift effect
+, Lift(..)
+, sendM
+, liftWith
+  -- * Re-exports
+, Algebra
+, Has
+, run
+) where
+
+import Control.Concurrent (ThreadId)
+import Control.Effect.Lift
+import Control.Exception hiding
+  ( throwIO
+  , ioError
+  , throwTo
+  , catch
+  , catches
+  , Handler
+  , catchJust
+  , handle
+  , handleJust
+  , try
+  , tryJust
+  , evaluate
+  , mask
+  , mask_
+  , uninterruptibleMask
+  , uninterruptibleMask_
+  , getMaskingState
+  , interruptible
+  , allowInterrupt
+  , bracket
+  , bracket_
+  , bracketOnError
+  , finally
+  , onException
+  )
+import qualified Control.Exception as Exc
+import Prelude hiding (ioError)
+
+-- | See @"Control.Exception".'Exc.throwIO'@.
+--
+-- @since 1.0.0.0
```

Comment:
All of the doc comments follow the same pattern. I don’t think we should attempt to document `Control.Exception` (or rather, if we do, we should do so _in_ `Control.Exception`), but we should definitely link to those docs.

🎩 @lexi-lambda for the haddock trick: the haddocks say “See `Control.Exception.throwIO`.”, with `Control.Exception` linking to the module and `throwIO` linking to the function. This is IMO _much_ better than just showing “See `throwIO`.”

Reply:
This is clever!